### ISSUE TITLE
Icon should not overtake aria-label

### ISSUE THREAD

**Issue by CezCz** [adobe/coral-spectrum#23]
### Expected Behavior

Developer can setup aria-label or aria-labelledby to anything in an Icon.
This is an extension of [https://github.com/adobe/coral-spectrum/issues/20](url)

### Actual Behavior

Coral forces usage of autogenerated aria-label.

### Reproduce Scenario (including but not limited to)

Run:

#### Steps to Reproduce

1. Run the code above.
2. Inspect that aria-label is "Add"
3. Expected value for aria-label should be "My aria label"

#### Browser name/version/os (ie Chrome Version 62.0.3202.94 (Official Build) (64-bit) MacOS)
Irrelevant

#### Coral Spectrum version

4.1.0

#### Sample Code that illustrates the problem (use the Playground if possible)

https://opensource.adobe.com/coral-spectrum/dist/playground/#?livereload=true&screen=vertical&code=MTczLDE0NCwxNzcsMTQsMTk0LDQ4LDEyLDY4LDI0NywxMjYsMTMzLDIyOSwxNTcsMTAyLDcxLDEwNSwzNywxMTgsMjE2LDg5LDIxMSwxOTYsMjA4LDY0LDEwNCwzNCwyMTksMjksMjAyLDIxNSwyMTEsNTQsNjYsMjUyLDAsMTM5LDEyNSwzOSw4OSwyMzksMTE2LDE4MiwzNSwxODUsMjA4LDU1LDAsNTQsMTk3LDIzMyw5LDc2LDE2OSw2NywyMDksMzcsMTQ1LDE0MCw2OCwxMzgsNDgsNTAsMjIxLDU4LDI4LDg1LDEzOSwyOCwxNDEsMjAxLDEzMywzOCwyMDEsNTEsMTIzLDEwNiw5MywyMDAsMywxODEsNjIsMTkxLDE0MCwyMDcsMjM2LDIxMCw2NSwxMCwxMjEsMjI5LDI0OSwxMDEsNjYsMjAsNTMsMzcsMTg1LDIyOSwyMDYsMTIxLDE1OCw2NiwyNDUsOTQsMTY0LDIyMiwxODEsMTcxLDY2LDE3OSw3LDEzOCwyMzEsODgsMjAsMTMyLDI1MywzMSwyLDMwLDk1LDI1NCw2NywxNzYsMTgzLDE2NiwxNzgsMjUxLDE5OCwxNTQsMjE4LDIwNywxNCw1Nyw0NCwyMzUsMTcwLDE2OCwyMzIsMjQzLDQsMjE5LDIzMiwyMDgsMTMzLDEyOCwxNjAsODEsMTksMTE3LDEyMCwyMTgsMTgwLDE5NiwyNDcsNDIsMTc1LDIxNSw1MSwyLDU2LDE0MiwyMzgsMTQ0LDIyMCwxNzYsNjEsMjI5LDE3OCwyMzYsMjIsMTE4LDE4Nyw2OSwyNTIsODAsMTI1LDI0Mywx&

### Proposed steps

1. Add parameter to coral icon, to ignore aria-label generation when not necessary.
2. Mark _updateAltText as deprecated, provide warning about future removal of aria-label generation in coral

**Comment by CezCz** [comment]
Closed by a mistake
